.github/workflows/CI.yml

```diff
@@ -184,9 +176,15 @@ jobs:
         shell: bash
         run: |
           # determine version
-          GIT_LAST_TAG=${{ steps.get_previous_tag.outputs.tag }}
-          GIT_NEXT_TAG=${{ steps.get_next_semver_tag.outputs.patch }}
-          echo "GIT_LAST_TAG=${GIT_LAST_TAG}, GIT_NEXT_TAG=${GIT_NEXT_TAG}"
+          npx standard-version@^9.3.1 \
+            --skip.commit \
+            --skip.tag \
+            --skip.changelog
+
+          GIT_NEXT_TAG=$(cat VERSION.txt)
+          git checkout HEAD -- VERSION.txt
+          
+          echo "GIT_NEXT_TAG=${GIT_NEXT_TAG}"
```

Comment:
those changes should also make it into https://github.com/keptn/gh-automation/blob/main/.github/workflows/prepare-ci.yml

Reply:
I am not so sure, they only work with an annotated changelog according to conventional changelog rules. I don't know if all integration use that

Reply:
So far only a couple integrations use the prepare-ci step from keptn/gh-automation, and those already use `standard-version` package ;)

Reply:
cool, then i will add it there as well

Reply:
this still needs the post-changelog script that i implemented to update the version.txt file with the new version string and specific configuration and such. I still think it's too specialized for keptn/keptn specifically

Reply:
ok